Class-6-(Inheritance).md

```diff
@@ -0,0 +1,19 @@
+# Class 6: Inheritance
+
+## In the prep work this week, the students learned
+1. What inheritance is and how programmers can use it.
+1. What abstract classes and methods are.
+1. How to formally diagram classes and the relationships between classes.
+
+## Announcements
+1. 
+
+## Large Group Time (Instructor)
+
+### Class 6 Topics That Require Careful Attention
+1. Inheritance is a topic that can seem very intuitive to some and very confusing to others. Be mindful of this gap and encourage discussion in pairs as short lecture activities to help students hear other perspectives on why programmers use inheritance.
```

Comment:
this is helpful! the second sentence reads a little long though. can you break it up into two?

Reply:
Done! Thanks!